dash_bio/NeedlePlot.py

```diff
@@ -5,30 +5,32 @@
 
 class NeedlePlot(Component):
     """A NeedlePlot component.
-
+The Needle Plot component is used to display large datasets
+containing categorical or numerical data. The lines and markers in
+the plot correspond to bars in a histogram.
 
 Keyword arguments:
 - id (string; optional): The ID of this component, used to identify dash components
 in callbacks. The ID needs to be unique across all of the
 components in an app.
-- mutationData (optional): . mutationData has the following type: dict containing keys 'x', 'y', 'mutationGroups', 'domains'.
+- mutationData (optional): The data that are displayed on the plot.. mutationData has the following type: dict containing keys 'x', 'y', 'mutationGroups', 'domains'.
```

Comment:
Looks like an extra `.` after the word plot here.

Reply:
Ugh, this looks like it's a problem with the Dash class generator. Period only shows up after the prop description if it's an object. I'll fix this just to get it to work now, but will need to make a PR in the Dash repo to fix the autogeneration.

Reply:
I sympathize, this has been the week of `dash-generate-components` for me.